FestivalAttendeeThread.java

```diff
@@ -0,0 +1,38 @@
+package concurrency;
+
+
+import java.util.LinkedList;
+
+public class FestivalAttendeeThread implements Runnable {
+
+
+    private LinkedList<TicketType> ticketList;
+
+    public FestivalAttendeeThread(LinkedList<TicketType> ticketList) {
+        this.ticketList = ticketList;
+    }
+
+    @Override
+    public void run() {
+
+//        for (int i = 0; i < 20; i++) {

```

Comment:
remove commented code, the logic should be to create 20 festivalAttendeeThreads (so basically for with thread, not a thread with a for)